content/docs/visualization/template_examples.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Template examples
+sort_rank: 4
+---
+
+# Template examples
+
+Prometheus has support for templating in both the summary/description field of
+alerts and also exposed as html endpoints. Templates have the ability to run
```

Comment:
I'd phrase it:

Prometheus supports templating in the summary and description fields of alerts, as well as in served console pages.